src/Workspaces/Core/Portable/Shared/Extensions/SyntaxGeneratorExtensions.cs

```diff
@@ -526,6 +526,9 @@ private static DeclarationModifiers GetOverrideModifiers(ISymbol symbol)
                     method: overriddenMethod,
                     accessibility: overriddenMethod.ComputeResultantAccessibility(newContainingType),
                     modifiers: modifiers,
+                    // There is no need for an override to have same parameters attributes as the original method.
```

Comment:
I worry this is a bit strong of a statement that there's _no_ need -- how do we know there aren't some reflection-based systems out there that are depending on this?

Which attributes triggered this original question?

Reply:
We already do not copy attributes of methods and attributes of properties. By mistake (my opinion), we copy attributes of method parameters and property parameters. We just should stop doing this.

---
In reply to: [290869693](https://github.com/dotnet/roslyn/pull/36163#discussion_r290869693) [](ancestors = 290869693)